### ISSUE TITLE
h3 tags inside p tags on thank you page

### ISSUE THREAD

**Issue by jonathan-dejong** [eventespresso/event-espresso-core#149]
Heyo,

Trying to modify the thank you page using `FHEE__thank_you_page_overview_template__order_conf_desc`.

I checked the source code and it looks like this:
```
<p class="bigger-text">
			<?php
			echo apply_filters(
				'FHEE__thank_you_page_overview_template__order_conf_desc',
				sprintf(
					$order_conf_desc,
					'<h3 class="">',
					'</h3>',
					'<br />'
				)
			);
			?>
			</p>
```
Seems a bit strange to me to wrap a h3 inside a p tag. It results in this kind of markup:
```
<div class="extra-padding-sides">
			<p class="bigger-text">
			</p><h3>Nu är det klart!</h3></br>Vi har registrerat din anmälan och skickat en bekräftelse till dig via e-post. Längre ned på den här sidan kan du se och göra ändringar i din anmälan.			<p></p>
						<div class="jst-rght">
				<a class="ee-button ee-roundish indented-text big-text" href="http://kfsk.wordpress.dev/registrering-slutford/?ee=launch_invoice&amp;id=1-682b4b54d914d55b5d08cbc071479d10&amp;receipt=true"><span class="ee-icon ee-icon-PDF-file-type"></span>Hämta beställningsbekräftelse/kvitto</a>
			</div>
</div>
```

Its a mess!

**Comment by mnelson4** [comment]
sorry to change the amusing ticket title, but I think this is a bit more descriptive ;). But I think it's a good find

**Comment by tn3rb** [comment]
the filter was probably moved from somewhere else... so no drunkenness involved... just copy pasta

**Comment by tn3rb** [comment]
here's a fix you can test:
https://github.com/eventespresso/event-espresso-core/tree/BUG-9540-thnx-page-html

**Comment by tn3rb** [comment]
going to change that fix actually, so that the translated strings don't change

**Comment by joshfeck** [comment]
The fix was included in the last release update (4.8.39)
